awscli/examples/cloudwatch/put-insight-rule.rst

```diff
@@ -0,0 +1,34 @@
+**To create a contributor insights rule**
+
+The following ``put-insight-rule`` example creates a Contributor Insights rule named ``VPCFlowLogsContributorInsights`` in the specified account. If the command succeeds, no output is returned. ::
```

Comment:
nit: Usually the comment that there is no output is after the command with the text "This command produces no output."

Reply:
Updated the suggestion.